src/components/pages/Theme/Theme.jsx

```diff
@@ -535,7 +741,7 @@ export const Theme = () => {
       <div className='section'>
         <Segment padded>
           <Header as='h2' dividing>
-            Table
+            Table - Semantic UI
```

Comment:
I think we can keep the title as is, to indicate it is our default, Semantic table.